dev/docker-compose.yml

```diff
@@ -116,6 +112,7 @@ services:
 
 
 volumes:
+  pulp: {}
   pulp_artifact: {}
```

Comment:
Is `pulp_artifact` still used / mapped above?

Reply:
It is not used now, removing.